src/constants.js

```diff
@@ -18,7 +18,13 @@ export const DIMENSION_KEYS = {
   total: "total",
 };
 
-const DIMENSIONS = {
+export const DATA_KEY_TRANSLATIONS = {
+  [DIMENSION_KEYS.gender]: "gender",
+  [DIMENSION_KEYS.age]: "age_bucket",
+  [DIMENSION_KEYS.race]: "race_or_ethnicity",
+};
```

Comment:
The keys in the provided data are a bit different than the ones we work with internally so this mapping provides a "translation" between the two worlds.